src/components/Header/HeaderLogo/HeaderLogo.jsx

```diff
@@ -0,0 +1,12 @@
+import './HeaderLogo.css';
+import LogoImg from '../../../../public/img/VeggyLogo.png';
+
+export default function HeaderLogo() {
+    return (
+        <div className="headerLogo">
+            <a href="#">
```

Comment:
Чому тут не поправив? Я ж робив зауваження, що не треба ставити заглушку на ссилку. Ссилка на логотип повинна вести на ГОЛОВНУ сторінку сайта. Так прийнято. 

```suggestion
            <a href="/">
```

Чому не виправив?